src/main/java/com/theironyard/war/Game.java

```diff
@@ -0,0 +1,130 @@
+package com.theironyard.war;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by EddyJ on 6/23/16.
+ */
+public class Game {
+    private Player player1;
+    private Player player2;
+
+    public Game(Player player1, Player player2) {
+        this.player1 = player1;
+        this.player2 = player2;
+    }
+
+    /**
+     * The method is going to set up each players hand of 26 random cards from the standard deck of 52 cards.
+     */
+    public void setUpHands() {
+        Deck deck = new Deck();
+        deck = Deck.createStandardDeck();
+        deck.shuffle();
+        for (int i = 1; i <= 26; i++) {
+            player1.hand.add(deck.dealCards());
```

Comment:
This is bad.  You should not be accessing the player's hand.  The player needs to be in control of the hand.  Add methods that allow the player to take the card and then have the player add that card to the hand.